client/my-sites/plugins/plugin.jsx

```diff
@@ -113,9 +116,8 @@ export default React.createClass( {
 
 	refreshSitesAndPlugins( nextProps ) {
 		this.setState( this.getSitesPlugin( nextProps ) );
-
 		// setTimeout to avoid React dispatch conflicts.
-		setTimeout( () => {
+		this.pluginRefreshTimeout = setTimeout( () => {
```

Comment:
Hmmm...that makes sense, thanks for the clear explanation :)

I have another question :p, can't we set the title at the controller? 
I think it could be useful for future cases if we return the `timeoutID` in https://github.com/Automattic/wp-calypso/blob/a7478174b0441760c332fcb4b8c454f5b8d73dfc/shared/lib/screen-title/actions.js#L16 and clear that one instead of creating a new one.